VirtoCommerce.OrderModule.Test/WorkflowControllerTest.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using CacheManager.Core;
+using Moq;
+using VirtoCommerce.Domain.Commerce.Model.Search;
+using VirtoCommerce.Domain.Customer.Model;
+using VirtoCommerce.OrderModule.Core.Models;
+using VirtoCommerce.OrderModule.Core.Services;
+using VirtoCommerce.OrderModule.Data.Model;
+using VirtoCommerce.OrderModule.Data.Repositories;
+using VirtoCommerce.OrderModule.Data.Services;
+using VirtoCommerce.Platform.Core.Assets;
+using VirtoCommerce.Platform.Core.Common;
+using VirtoCommerce.Platform.Data.Infrastructure;
+using Xunit;
+
+namespace VirtoCommerce.OrderModule.Test
+{
+    [CLSCompliant(false)]
+    [Trait("Workflow", "WF")]
+    public class WorkflowControllerTest
```

Comment:
@vinhvoniteco 
1. The class name should be `WorkflowServiceTest`, not controller
2. Seem it lacks of inheriting `TestBase`